daisy/disk.go

```diff
@@ -188,7 +189,7 @@ func (dr *diskRegistry) detachHelper(dName, iName string, isAttached bool, s *St
 		return nil
 	}
 
-	pre := fmt.Sprintf("step %q cannot detach disk %q from instance %q", s.name, dName, iName)
+	pre := fmt.Sprintf("step %q cannot detach disk with device name = %q from instance %q", s.name, dName, iName)
```

Comment:
minor: remove `=`. If you want to highlight it, consider putting it in quotes.

Also, dName can be diskName or deviceName. I thought you wanted to get rid of this var name?

Reply:
done